src/_pytest/terminal.py

```diff
@@ -1007,6 +1007,14 @@ def _folded_skips(skipped):
     return values
 
 
+def _match_plurality(count, key):
```

Comment:
I think it makes more sense to always pass the key as singular, and add `s` as necessary:

```python
def _make_plural(noun, count):
   return noun + "s" if count != 1 else noun
```

This of course requires a change in the key themselves, but I believe this to be more elegant. What do you think?

Reply:
Yes, `_make_plural` is an easier-to-understand name, thanks. I've kept the guard in though as we wouldn't want to add an 's' to the end of 'skipped', for example.

Reply:
> This of course requires a change in the key themselves, but I believe this to be more elegant. What do you think?

How backwards-compatible would that be (warnings -> warning) for e.g. plugins?

Reply:
It might cause minor breakages for plugins testing things with `testdir`, depending on the test, good point.

To be on the safe side we should target `features` instead and get this out in `5.3`. @MarcoGorelli could you rebase on `features` and change the target branch of the PR? Thanks!

Reply:
> changed the base branch from master  to features

@marcogorelli
You also need to rebase it on features then (and force-push).

Reply:
Do not add a commit that removes the changes (https://github.com/pytest-dev/pytest/pull/5990/commits/31af9692127612786adcec01f1379be3cca16e35), but rebase it so that only your commit(s) are there.

Reply:
Ok, I think I've got it now (sorry, bit of a git-rebase noob here), thanks to you all for your time!

Reply:
Better.. but the commit message needs adjustments.. ;)